### PR TITLE
Updated Translations for zh_hans

### LINKED ISSUE #2344: Translate App Strings to Chinese (Simplified zh_hans)
### User Story

As a Chinese language speaker, I want to be able to interact with Status in my native language.

### Description

*Type*: Feature

*Summary*: We need someone fluent (and eloquent) in Chinese to translate all the latest strings in Status from English. We would like to invite our community members to take on these tasks in order
a) To become a bit more acquainted with the Status repos and perhaps wander further into the codebase and set of issues we need help with, earning yourself Open Bounties all the way down this rabbit hole, and
b) To make sure that the product we deliver has been built by the same community that uses it, in a virtuous feedback loop of increasing understanding and value-generation.

#### Expected behavior
The Chinese translation ought to be as up-to-date as the English one.

#### Actual behavior
We have not yet updated it, preferring to leave this open as an opportunity for our community to get involved again.

### Solution

*Summary*: 

- The file [here](https://github.com/status-im/status-react/blob/develop/src/status_im/translations/zh_hans.cljs) needs to be updated to match the new English one [here](https://github.com/status-im/status-react/blob/develop/src/status_im/translations/en.cljs).
- How you translate is a matter of your preference. There were significant updates to the English file for the 0.9.12 release, with lots of strings both add and REMOVED. We need you to make sure that the files are exact mirrors of each other, except in different languages.
- It might be easier to simply take the new English file and translate from scratch rather than going through the current translated files line by line (though we leave this up to you).

### Additional Information

* Status version: 0.9.12+

There are bounties available for these translations. Please see the QR codes that will be added below after the DevCon 3 announcement.

### PR DESCRIPTION
**Pull request by leoscku**
fixes #2344

### CODE REVIEW

**Comment by andytudhope** [comment]
Amazing stuff, thank you so much! You're a rockstar.

File: src/status_im/translations/zh_hans.cljs [hunk 1]
```diff
@@ -11,19 +11,22 @@
    :search-for                            "搜索..."
    :cancel                                "取消"
    :next                                  "下一个"
+   :open                                  "开启"
+   :description                           "Description"
```
Comment (ninadou) [inline src/status_im/translations/zh_hans.cljs:15]: Why is “Description” still an English term? I think it should be "描述"

File: src/status_im/translations/zh_hans.cljs [hunk 2]
```diff
@@ -11,19 +11,22 @@
...
+   :url                                   "连结"
```
Comment (ninadou) [inline src/status_im/translations/zh_hans.cljs:16]: I think it should be "链接" here.

File: src/status_im/translations/zh_hans.cljs (hunk 1 above)
Comment (ninadou) [inline src/status_im/translations/zh_hans.cljs:15]: It should be "描述" here.

File: src/status_im/translations/zh_hans.cljs [hunk 3]
```diff
@@ -182,52 +182,35 @@
...
-   :group-chat-name                       "聊天名称"
-   :empty-group-chat-name                 "请输入名称"
-   :illegal-group-chat-name               "请选择其它名称"
    :new-group                             "新增群組"
    :reorder-groups                        "重新排序群組"
-   :group-name                            "群組名稱"
    :edit-group                            "編輯群組"
    :delete-group                          "刪除群組"
    :delete-group-confirmation             "此群組將從你的群組中移除。 這不會影響聯絡人"
    :delete-group-prompt                   "這不會影響聯絡人"
```
Comment (ninadou) [inline src/status_im/translations/zh_hans.cljs:211]: I think these traditional Chinese should be deleted too.

File: src/status_im/translations/zh_hans.cljs [hunk 4]
```diff
@@ -92,15 +90,13 @@
    :remove-from-contacts                  "从联系人中删除"
    :start-conversation                    "开始对话"
    :send-transaction                      "发送交易"
-   :share-qr                              "分享QR码"
-   :error-incorrect-name                  "请选择其它名称"
-   :error-incorrect-email                 "电子邮件不正确"
+   :testnet-text                          "你正在 {{testnet}} 测试网. 请不要发送真实 ETH 或 SNT 到您的位址"
```
Comment (ninadou) [inline src/status_im/translations/zh_hans.cljs:93]: "你正在 {{testnet}} 测试网. 请不要发送真实 ETH 或 SNT 到您的位置"

File: src/status_im/translations/zh_hans.cljs [hunk 5]
```diff
@@ -182,52 +182,35 @@
...
+   :sharing-copy-to-clipboard-coordinates "复制经纬度"
 
    ;new-group
-   :group-chat-name                       "聊天名称"
-   :empty-group-chat-name                 "请输入名称"
-   :illegal-group-chat-name               "请选择其它名称"
    :new-group                             "新增群組"
    :reorder-groups                        "重新排序群組"
-   :group-name                            "群組名稱"
    :edit-group                            "編輯群組"
```
Comment (ninadou) [inline src/status_im/translations/zh_hans.cljs:208]: "编辑群組"

File: src/status_im/translations/zh_hans.cljs [hunk 6]
```diff
@@ -182,52 +182,35 @@
...
    :delete-group                          "刪除群組"
    :delete-group-confirmation             "此群組將從你的群組中移除。 這不會影響聯絡人"
```
Comment (ninadou) [inline src/status_im/translations/zh_hans.cljs:210]: "此群组将从你的群组中移除。这不会影响联络人"

File: src/status_im/translations/zh_hans.cljs [hunk 7]
```diff
@@ -239,76 +222,144 @@
...
+   :paste-json-as-text                    "贴上 JSON 成文字"
+   :paste-json                            "贴上 JSON"
+   :specify-rpc-url                       "注明 RPC URL"
+   :edit-network-config                   "改变网路设置"
+   :connected                             "已连结"
+   :process-json                          "处理 JSON"
+   :error-processing-json                 "JSON 处理错误"
+   :rpc-url                               "RPC URL"
+   :remove-network                        "移除网路"
+   :network-settings                      "网路设置"
```
Comment (ninadou) [inline src/status_im/translations/zh_hans.cljs:360]: "网络设置"

**Comment by ninadou** [comment]
Hi, I've changed some terms as they're traditional Chinese or some other minus problems.

**Comment by oskarth** [comment]
Hi @liontomic! Any chance you can update your PR with the changes @ninadou suggested?

**Comment by andytudhope** [comment]
zh_hans translation accepted from daSheng

**Pull request closed without merging**